modules/apps/analytics/analytics-settings-rest-impl/src/main/java/com/liferay/analytics/settings/rest/internal/resource/v1_0/SiteResourceImpl.java

```diff
@@ -116,7 +116,9 @@ private LinkedHashMap<String, Object> _getParams() {
 	@Reference
 	private Portal _portal;
 
-	@Reference
-	private SiteDTOConverter _siteDTOConverter;
+	@Reference(
+		target = "(dto.class.name=com.liferay.portal.kernel.model.Group)"
+	)
+	private DTOConverter<Group, Site> _siteDTOConverter;
```

Comment:
@MrJohn1911 There are two DTOConverter registered with "dto.class.name=com.liferay.portal.kernel.model.Group", so this is wrong.